src/ui/forum.html

```diff
@@ -7,8 +7,9 @@
 <head>
 
     <meta charset="utf-8" />
-    <title>Button Men Online &mdash; Forum</title>
+    <title>Button Men Online</title>
```

Comment:
Indeed! And I thought having some header was better than no header in case there was an error reaching the server.